pkg/constants/constants.go

```diff
@@ -100,6 +100,15 @@ const (
 	// If set to "true", e2e-tests installer will configure master/control plane nodes as schedulable
 	ENABLE_SCHEDULING_ON_MASTER_NODES_ENV = "ENABLE_SCHEDULING_ON_MASTER_NODES"
 
+	// A gitlab token is required to run tests against gitlab.com. The token need to have permissions to the given github organization.
+	GITLAB_TOKEN_ENV string = "PAC_GITLAB_TOKEN" // #nosec
+
+	// The gitlab URL used to run integration reporting e2e tests against
+	GITLAB_URL_ENV string = "PAC_GITLAB_URL" // #nosec
+
+	// For PaC Gitlab tests required project ID
+	GITLAB_PRIJECT_ID string = "PAC_PRIJECT_ID"
+
```

Comment:
@psturc @dheerajodha 
I'm sure that the Token variable we want to add it to the CI vault, 
for the two other ones  `GITLAB_URL_ENV` and `GITLAB_PRIJECT_ID` should I use them as constant in code and not to be added to the vault ?

Reply:
for now I've added those three env variables to vault to make progress with this test, 
I will modify once we decide .